python/translit/converters/translit_cyr_taras_to_lat.py

```diff
@@ -6,7 +6,7 @@
 # Since the script is optimized for Nasha Niva's requirements,
 # there are some places that will be reworked to make the converter more generic.
 
-def convert(text):
+def convert(text: str) -> str:
```

Comment:
Not sure about static typing. according to this doc: https://docs.python.org/3/library/typing.html feature was introduced in Python 3.5. Does that mean library will require python 3.5+ ?

Reply:
Hm. Good point. I kinda assumed everyone is on python3 now. Because at Google we moved to 3+ :) Anyway, adding typing indeed requires some considaration. Undid typing.